### PR TITLE
Update Queues to bootstrap 5 and skip controls in qPCR pools

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

File: status/queues.py [hunk 1]
```diff
@@ -47,12 +52,18 @@ def get(self):
...
+                    if art.name in qpcr_control_names:
+                        continue
+                    try:
                         library_type = art.samples[0].project.udf.get("Library construction method", 'NA')
-                        try:
-                            runmode = art.samples[0].project.udf['Sequencing platform']
-                        except KeyError:
-                            runmode = 'NA'
+                    except:
+                        import pdb; pdb.set_trace()
```
Comment (alneberg) [inline status/queues.py:62]: That's not supposed to be there... :)

Comment (aanil) [reply status/queues.py]: Oops! Removed it now.

**Review by alneberg — APPROVED** [review]
Looks good! Will need some clean-up of the queues.py file before I press merge.

**Comment by alneberg** [comment]
Didn't see the last commit before I sent the review. All good now.

**Pull request merged**